bench.js

```diff
@@ -227,6 +229,7 @@ var PolkadotRuntimeBenchmarkConfigs = {
       "--features=runtime-benchmarks",
       "--",
       "benchmark",
+      "pallet",
```

Comment:
It could also be needed in Lines 328 and 419. Although I dont know what they do :laughing:   
@joao-paulo-parity maybe you know.

Reply:
From what I understand those are custom "run whatever" commands which could have a different subcommand other than `pallet`, so I think they can stay as they are. I don't know who's using them anyhow, perhaps @shawntabrizi can shed some light here.

In any case that should not block this PR and we can add it later if necessary.